pipelines/tekton/azureml-container-pipeline/model-upload/Makefile

```diff
@@ -0,0 +1,32 @@
+# Used for uploading local model files to the cluster through a 
+# PVC. Use create to create the pod and PVC needed, then use
+# upload model to upload a local file the PVC that the pod
+# is mounted to
+
+SIZE=1G
+PVC=model-upload-pvc
```

Comment:
When adding information to the README about the targets here, don't forget to document the variables and how to override them.

Reply:
Done 👍